### PR TITLE
Website Updates

### PR DESCRIPTION
**Pull request by eteridvalishvili**
CDM Website Update - August 

- [x] Meeting recording of Brian Lynns release schedule from Steering WG: https://resources.finos.org/category/projects-sigs/common-domain-model-cdm?projects-sigs=common-domain-model-cdm&c=cG9zdDo5OTE3MDQ%3D
- [x] Update “versioned??” Contribution.md page on the website to match the original
- [x] Update Governance.md 
    - [x] Section 4 language
    - [x] Section 6.0 include a link to the PowerPoint deck (as a PDF) shared by Brian Lynn

Add a Demo page 
- [x] Add OTC Digital Demo: https://resources.finos.org/znglist/real-world-project-demos?c=cG9zdDo5OTE1NzA%3D
- [x]  Add Barcelona Hackathon Demo: https://www.youtube.com/watch?v=N9C2ZokZAV4
- [x] Add ISDA DRR CDM Training Sessions Demo: https://www.youtube.com/watch?v=WaXR5uX-R8s
- [x] Add How to get started session: https://resources.finos.org/category/projects-sigs/common-domain-model-cdm?projects-sigs=common-domain-model-cdm&c=cG9zdDo5OTE2Mjg%3D

### CODE REVIEW

File: docs/home.mdx
Comment (chrisisla) [inline docs/home.mdx:25]: think you need to remove the ".md" from the end for this to work properly

File: website/versioned_docs/version-5.13.0/contribution.md
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/contribution.md:34]: "new new"

File: website/versioned_docs/version-5.13.0/home.mdx
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/home.mdx:25]: think you need to remove the .md suffix

File: website/versioned_docs/version-5.13.0/cdm-demos.md
Comment (chrisisla) [inline website/versioned_docs/version-5.13.0/cdm-demos.md:7]: open quotes but no close quotes?

File: docs/cdm-demos.md
Comment (chrisisla) [inline docs/cdm-demos.md:7]: open quotes no close quotes?

**Review by chrisisla — CHANGES_REQUESTED** [review]

**Review by dshoneisda — APPROVED** [review]

**Review by chrisisla — APPROVED** [review]

**Comment by eteridvalishvili** [comment]
Hi @finos/cdm-maintainers Could you please help me understand why this PR is failing? I can't seem to figure it out.

**Comment by eteridvalishvili** [comment]
Hi @tomhealey-icma When you have a chance could I get your feedback on how I can fix this PR? Thank you in advance!

**Comment by eteridvalishvili** [comment]
Hi @chrisisla and @dshoneisda Could you please review and merge this PR if everything looks good? Thank you in advance!

**Review by dshoneisda — APPROVED** [review]

**Pull request merged**
